src/book/02-system/02-subsystems/06-behavior.mdx

```diff
@@ -5,3 +5,27 @@ title: Behaviour
 description: Behaviour and how it works in the NUbots codebase.
 slug: /system/subsystems/behaviour
 ---
+
+The behaviour subsystem includes walk planning, soccer strategies, get up detection, head behaviour, script running and tuning, and kick behaviour.
+
+# Script Runner
+
+[ScriptRunner](https://github.com/NUbots/NUbots/tree/master/module/behaviour/tools/ScriptRunner) is a module in the behaviour subsystem. It takes the name of one or more script files as arguments and attempts to run the scripts. It does not take file paths, only the file name/s of the script/s to execute. Any role with the ScriptRunner module must also contain the [ScriptEngine](/system/subsystems/motion#script-engine) module.
+
+When ScriptRunner is run, it stores the script file names passed to it as a vector of strings. When the middle button on the robot is pushed, the scripts are executed in order of appearance, one after the other. ScriptRunner also provides an option to set a delay before the execution of the first script, as well as a delay between the execution of each script.
```

Comment:
Is the middle button the green button? I have never heard someone call it the middle button (although the button press event is "MiddleButtonDown" or something, so maybe this is ok)

Reply:
I think it is. The red button is on one side of it and on the other side is an empty spot that looks like a button.

Reply:
It is, I use both terms but if it's someone who might not be familiar with the buttons, I'll say green bc it's easier to locate since it's the only green thing on the robot.